src/compas_cgal/straight_skeleton_2.py

```diff
@@ -31,10 +64,10 @@ def create_interior_straight_skeleton(points) -> PolylinesNumpy:
     if not TOL.is_allclose(normal, [0, 0, 1]):
         raise ValueError("The normal of the polygon should be [0, 0, 1]. The normal of the provided polygon is {}".format(normal))
     V = np.asarray(points, dtype=np.float64)
-    return straight_skeleton_2.create_interior_straight_skeleton(V)
+    return graph_from_skeleton_data(*straight_skeleton_2.create_interior_straight_skeleton(V))
```

Comment:
just pointing out that the user can still "only" get a graph as a result and no longer has the option to use the low level output.

perhaps you could add a parameter ``as_graph`` and only convert to a graph if that parameter is set to ``True``, which could even be the default...